opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/sitecustomize.py

```diff
@@ -119,7 +119,7 @@ def initialize():
         logger.exception("Failed to auto initialize opentelemetry")
     finally:
         environ["PYTHONPATH"] = sub(
-            r"{}{}?".format(dirname(abspath(__file__)), pathsep),
+            fr"{dirname(abspath(__file__))}{pathsep}?",
```

Comment:
is this valid?

Reply:
nevermind, just checked.
```python
>>> a = 1
>>> fr"a={a}"
'a=1'
>>>
```

Reply:
yup :) been testing all these changes and im as surprised as you are :D